apps/remix-ide/src/walkthroughService.js

```diff
@@ -41,15 +41,15 @@ export class WalkthroughService extends Plugin {
       {
         element: document.querySelector('#udappRecorderUseLatest'),
         title: 'Transactions Recorder',
-        intro: 'If set the recorder will run transactions using the latest compilation result.',
+        intro: 'If selected the recorder will run transactions using the latest compilation result.',
         tooltipClass: 'bg-light text-dark',
         position: 'right',
         highlightClass: 'bg-light border border-warning'
       },
       {
         element: document.querySelector('#udappRecorderSave'),
         title: 'Transactions Recorder',
-        intro: 'Once there is a Once one or a few transactions have been executed from Remix, click this button to save these transactions as a scenario file.',
+        intro: 'Once there is a one or a few transactions have been executed from Remix, click this button to save these transactions as a scenario file.',
```

Comment:
The content here should be "Once one or more transactions have been executed from Remix, click this button to save these transactions as a scenario file."